nwcsafpps_runner/pps2018_runner.py

```diff
@@ -47,17 +47,19 @@
 from nwcsafpps_runner.utils import (SENSOR_LIST,
                                     SATELLITE_NAME,
                                     METOP_NAME_LETTER)
+
 from nwcsafpps_runner.publish_and_listen import FileListener, FilePublisher
 
 from nwcsafpps_runner.prepare_nwp import update_nwp
 
+
 #===============================================================================
 # import pdb
 # from config import get_config  # @UnresolvedImport @UnusedImport
 # from config import MODE  # @UnresolvedImport @Reimport
 # from config import CONFIG_FILE  # @UnresolvedImport @UnusedImport
 # from config import CONFIG_PATH  # @UnresolvedImport @UnusedImport
-# 
+#
 # from utils import ready2run, publish_pps_files  # @UnresolvedImport @UnusedImport
 # from utils import (get_sceneid, prepare_pps_arguments,              # @UnresolvedImport @UnusedImport
 #                    create_pps2018_call_command, get_pps_inputfile,  # @UnresolvedImport @UnusedImport
```

Comment:
Can these be removed or do they serve a purpose ?

Reply:
They do serve a big purpose for me. But if they are in the way I will figure out a different way to keep them.

Reply:
Removed in new PR